azure-pipelines.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       make --keep-going test
     displayName: 'Run rust tests'
   - script: |
-      echo let us wait until cargo fix or rustfix actually works make --keep-going clippy
+      make --keep-going clippy
     displayName: 'SKIP clippy linter'
```

Comment:
Update display name since you're no longer skipping?

Reply:
or just revert that one commit that did the change